pkg/data/components/TraceConverter.yaml

```diff
@@ -45,6 +45,22 @@ properties:
     validations: [int]
     default: 4318
     advanced: true
+  - name: Headers
+    summary: Headers to emit when sending HTTP traffic.
+    description: |
+      Sending data to a backend may require additional headers to be
+      configured. This properties supports sending a map of header keys and
+      values.
+    type: map
+    validations: []
```

Comment:
I think this should be an advanced property (in this component) because under normal use, they won't need to add anything to send the data to Refinery.

Reply:
think that will depend on whether or not a send key is configured on the honeycomb exporter. will merge like this for now and revised if needed next week